gp-includes/formats/format-android.php

```diff
@@ -10,6 +10,14 @@ class GP_Format_Android extends GP_Format {
 	public function print_exported_file( $project, $locale, $translation_set, $entries ) {
 		$this->exported = '';
 		$this->line( '<?xml version="1.0" encoding="utf-8"?>' );
+
+		$this->line( '<!--' );
+		$this->line( 'Translation-Revision-Date: ' . GP::$translation->last_modified( $translation_set ) . '+0000' );
+		$this->line( "Plural-Forms: nplurals={$locale->nplurals}; plural={$locale->plural_expression};" );
+		$this->line( 'Generator: GlotPress/' . GP_VERSION );
+		$this->line( 'Language: ' . $locale->english_name );
```

Comment:
I think we should make the language value consistent through all formats.
POMO uses the [ISO 639 codes](https://github.com/GlotPress/GlotPress-WP/blob/develop/gp-includes/formats/format-pomo.php#L89), Jed 1.x uses [the slug of a GP locale](https://github.com/GlotPress/GlotPress-WP/blob/develop/gp-includes/formats/format-jed1x.php#L59) and this PR proposes to use the English name for the others.

What about moving `GP_Format_PO::get_language_code()` to `GP_Format` so it can be used for all formats?

/cc: @swissspidy